chapters/sampling-randomization-power.tex

```diff
@@ -1,529 +1,788 @@
 %-----------------------------------------------------------------------------------------------
 
 \begin{fullwidth}
-Sampling and randomized assignment are two core elements of research design.
-In experimental methods, sampling and randomization directly determine
+
+In this chapter we will show how you can save a lot of time
+and increase the quality of your research by planning your data requirements
+based on your research design.
+There are many published resources about 
+the theories behind different research designs.
+This chapter will instead focus on how the design 
+impacts a project's data requirements.
+We assume you have a working familiarity
+with the research designs mentioned here.
+If needed, you can reference Appendix XYZ,
+where you will find more details 
+and specific references for each design.
+
+Planning data requirements is more than just listing the data your project will need. 
+Planning data requirements includes understanding how to
+link the different data sources your project will use.
+It also involves creating the tools your team can use
+to make sure that understanding is shared across your team.
+The first step of this process is to create master datasets 
+for every unit of observation in your study,
+and to create a data map, 
+detailing how the master datasets are interlinked.
+These master datasets will serve three key functions.
+First, they will list all of the units who are eligible for the study,
+and enable you to map data collected or received from the field to the research design.
+Second, in designs where your team has direct control over interventions or other field work,
+they will allow you to complete sampling and treatment assignment tasks
+before those tasks are implemented in the field.
+Finally, they will function as a single unambiguous location where all information 
+related to the implementation and validity of your research is stored,
+as well as all information needed to correctly identify any observation in any of your project's datasets.
+
+The second half of this chapter covers random sampling and assignment
+and the necessary practices to ensure that 
+these and other random processes are reproducible.
+Almost all research designs rely on a random component 
+for the results of the research to be a valid interpretation of the real world.
+This includes both how a sample is representative to the population studied,
+or how the counterfactual observations in experimental design are statistically indistinguishable
+from the treatment observations.
+The second half concludes with a discussion of power calculations and randomization inference,
+and how both are important tools to make optimal choices when planning data work.
+
+
+\end{fullwidth}
+
+%-----------------------------------------------------------------------------------------------
+
+\section{Translating research design to master data}
+
+In most projects, more than one dataset is needed to answer the research question.
```

Comment:
I see what you are saying and I am in general agreeing with you. But two points:
* I do not think that the fact that one dataset can have multiple tables, means that that you cannot have several datasets with one table each. It is up you you to define where you draw the line. I think it is perfectly fine to say that my survey data set is one dataset and my scraping data is a different dataset.
* While I think it is part of the book's purpose to push correct usage of terminology, it might be too much to push that strict definition of datasets. Agree worth discussing.

Reply:
@luizaandrade i believe you said you agreed with Kris -- if you do not think we need to discuss further, please resolve